js/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "lib/index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/vispy/jupyter_rfb.git"
+    "url": "https://github.com/jupyter_rfb/jupyter_rfb.git"
```

Comment:
This is not correct, it's still under the `vispy` org.

Reply:
i dont get your edit ....

Reply:
ok sorry i edit